partials/hero.hbs

```diff
@@ -2,30 +2,39 @@
 If the template in question has a background image, then we render responsive image styles
 for it, and apply those styles to the <header> tag.
 --}}
+{{!--
+If the template in question has a background image, then we render responsive image styles
+for it, and apply those styles to the <header> tag.
+--}}
 
 {{#if background}}
   {{#contentFor "herobackground"}}
-  <style>
-    .m-hero__picture {
-      background-image: url({{img_url background size='xl'}});
-    }
-  
-    @media(max-width: 1000px) {
+    <style>
       .m-hero__picture {
-        background-image: url({{img_url background size='l'}});
+        display: flex;
+        align-items: center;
+        justify-content: center;
       }
-    }
-  
-    @media(max-width: 600px) {
-      .m-hero__picture {
-        background-image: url({{img_url background size='m'}});
+      .m-hero__picture img{
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
       }
-    }
-  </style>
+    </style>
   {{/contentFor}}
 
   <section class="m-hero with-picture" data-aos="fade">
-    <div class="m-hero__picture {{#is "post"}}in-post{{/is}}"></div>
+    <div class="m-hero__picture {{#is "post"}}in-post{{/is}}">
+      <img class="post-image"
+           srcset="{{img_url background size="s"}} 300w,
+            {{img_url background size="m"}} 600w,
+            {{img_url background size="l"}} 1000w,
+            {{img_url background size="xl"}} 2000w"
+           sizes="(max-width: 600px) 600px, (max-width: 1000px) 1000px, 2000px"
+           src="{{img_url background size="l"}}"
+           alt="{{title}}"
+      />
+    </div>
 {{else}}
   <section class="m-hero no-picture {{#is "post"}}in-post{{/is}}" data-aos="fade">
-{{/if}}
+{{/if}}
```

Comment:
Would you mind to add a new line at the end of the file? 🙏

Reply:
Done!